tsconfig.packages.json

```diff
@@ -29,4 +32,4 @@
       "path": "./modules/web-demo"
     }
   ]
-}
+}
```

Comment:
We need a new line here? Maybe we need `.editorconfig` for the SDK?

Reply:
should have newlines at the end of every file, check your IDE settings if you're not seeing this on your changes